### DISCUSSION TITLE
how to change platform in presence

### DISCUSSION THREAD

**Discussion by aurkaxi** [Rapptz/discord.py#8433] [Q&A]
```python
discord.Streaming(platform="YouTube", name="name", game="well this does nothing", url="stream_url")
```

but still discord says "Live on twitch + (watch button)"
no other attributes are working but URL and Name only.

**Comment by DenverCoder1** [comment]
Seems like there's something not matching up either on Discord's end or Discord.py, but setting the `name` to `YouTube` will get you the "Live on YouTube" message.

```py
discord.Streaming(name="YouTube", url="https://www.youtube.com/...")
```

**Reply by aurkaxi** [reply]
Yeah it does work. But then while not viewing profile it will look "Streaming Youtube" no custom name ugh. Tried to workaround by `details="jfskdv"` but then youtube disappears. streaming jfskdv and in profile live on twitch again. 
so i will now work with name=youtube but not marking this as aanswer. have a nice day, sir.

**Reply by DenverCoder1** [reply]
I tried modifying the data sent directly to Discord and it seems like Discord does nothing with the details and state. So I think this is your only option if you want it to say Live on YouTube with the Streaming type.

As far as I can tell, Discord.py can't help you any more with this. You can possibly bring it up at https://github.com/discord/discord-api-docs/discussions instead to get Discord involved.

Here's an example raw activity payload in case you want to mention it in a Discord discussion `{'type': 1, 'name': 'YouTube', 'url': 'https://www.youtube.com/watch?v=example', 'state': 'Name of Game', 'details': 'Some Details'}`

![image](https://user-images.githubusercontent.com/20955511/188336056-74452f48-0dbf-461f-ab56-b2a232667fe9.png)
